src/components/CommunityProjectsCard/CommunityProjectCard.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Card } from "@shopify/polaris";
+import styles from "./CommunityProjectsStyles.module.css";
+
+export function CommunityProjectCard({ title, link, description }) {
+  return (
+    <div className={styles.singleCardWrapper}>
+      <Card
+        title={String(title)}
```

Comment:
Once we have a props interface, TS will know `title` is a string and we won't need to wrap it in `String()`

Reply:
Ahh ok I understand I'll try and do that. In general, should I be creating a props interface for all components? Passing the interface like ```(props: PropsInterface)``` should avoid the implicit any issue as well right? I'm a huge typescript noob sorry lol

Reply:
Any component that accepts props should have those props typed to an interface so as to prevent vars with the wrong datatypes from getting passed in.

So the common pattern is

```
interface Props {
  title: string;
  ...
}

export const MyComponent(props: Props) => {
...
```

Reply:
Got it. Added a props interface in 173c6ba